includes/Elements/Login_Register.php

```diff
@@ -6565,8 +6567,8 @@ protected function print_form_illustration($form_type = 'login') {
 	 * @param string $form_type the type of form. Available values: login and register
 	 */
 	protected function print_form_header( $form_type = 'login' ) {
-		$title    = ! empty( $this->ds["{$form_type}_form_title"] ) ?  esc_html__( wp_strip_all_tags( $this->ds["{$form_type}_form_title"] ), 'essential-addons-for-elementor-lite' )  : '';
-		$subtitle = ! empty( $this->ds["{$form_type}_form_subtitle"] ) ? HelperCLass::eael_wp_kses( $this->ds["{$form_type}_form_subtitle"] ) : '';
+		$title    = ! empty( $this->ds["{$form_type}_form_title"] ) ?  wp_strip_all_tags( $this->ds["{$form_type}_form_title"] )  : '';
```

Comment:
We need to check if __ works for settings value in the future.
For now, assuming the changes are good to go.

Reply:
`__()` may work but PCP suggest to use `esc_html__()`. Because `__()` just translate escape nothing